fenicsprecice/adapter_core.py

```diff
@@ -3,6 +3,7 @@
 """
 
 from fenics import SubDomain, Point, PointSource, vertices, FunctionSpace, Function, edges
+import fenics
```

Comment:
Why do you have these two imports here? Either `from fenics import everything_you_need` or `import fenics` and then `fenics.whatever_you_need_when_you_actually_need_it`